src/main/java/graphql/validation/ValidationErrorTypeInterface.java

```diff
@@ -0,0 +1,8 @@
+package graphql.validation;
+
+
+import graphql.PublicApi;
+
+@PublicApi
+public interface ValidationErrorTypeInterface {
```

Comment:
Mark interface for validation error type.

Reply:
I am happy to accept this PR - However I would like the name of this to be  `ValidationErrorClassification` (like how we have `ErrorClassification`